src/locales/en.json

```diff
@@ -74,6 +76,7 @@
   "Logout": "Logout",
   "More options": "More options",
   "New Batch": "New Batch",
```

Comment:
Remove `New Batch` from en.json if not used.

Reply:
Done sir!